__playwright-tests__/archiving-assets.spec.ts

```diff
@@ -1,5 +1,16 @@
 import { test, expect } from '../src';
 
+const externalImg =
```

Comment:
Can we read in one of the image files in `fixtures` to respond with instead of embedding this here?